docs/rst/fastdds_qos_profiles_manager_cli/cli_elements/cli_transport_descriptor_element.rst

```diff
@@ -1,4 +1,47 @@
+.. include:: ../../01-exports/roles.include
+
 .. _fastdds_qos_profiles_manager_cli_transport_elements:
 
 Transport Descriptor configuration access
 -----------------------------------------
+
+This section explains how to access the different configurable parameters in a transport descriptor profile.
+The scope of this section does not include explaining the usage of these specific parameters.
+Please, refer to `Fast DDS documentation <https://fast-dds.docs.eprosima.com/en/latest/fastdds/xml_configuration/transports.html>`_ for that information.
+
+The transport descriptor profile is accessed through the ``transport_descriptor`` element.
+The profile name is mandatory and must be passed within brackets.
```

Comment:
As this is the only exception in which the identifier works in a different way (it is not an attribute called `profile_name`), what do you think about adding some kind of clarification regarding the `transport_id` element? 

Would that be enough confusing to the reader that it does not make sense event to mention that?